module/VuFind/src/VuFind/Db/Entity/LoginToken.php

```diff
@@ -0,0 +1,360 @@
+     */
+    public function getToken(): string
+    {
+        return $this->token;
+    }
+
+    /**
+     * Set series string.
+     *
+     * @param string $series Series
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setSeries(string $series): LoginTokenEntityInterface
+    {
+        $this->series = $series;
+        return $this;
+    }
+
+    /**
+     * Get series string.
+     *
+     * @return string
+     */
+    public function getSeries(): string
+    {
+        return $this->series;
+    }
+
+    /**
+     * Set last login date/time.
+     *
+     * @param DateTime $dateTime Last login date/time
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setLastLogin(DateTime $dateTime): LoginTokenEntityInterface
+    {
+        $this->last_login = $dateTime->format('Y-m-d H:i:s');
+        return $this;
+    }
+
+    /**
+     * Get last login date/time.
+     *
+     * @return DateTime
+     */
+    public function getLastLogin(): DateTime
+    {
+        return DateTime::createFromFormat('Y-m-d H:i:s', $this->last_login);
+    }
+
+    /**
+     * Set browser details (or null for none).
+     *
+     * @param ?string $browser Browser details (or null for none)
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setBrowser(?string $browser): LoginTokenEntityInterface
+    {
+        $this->browser = $browser;
+        return $this;
+    }
+
+    /**
+     * Get browser details (or null for none).
+     *
+     * @return ?string
+     */
+    public function getBrowser(): ?string
+    {
+        return $this->browser;
+    }
+
+    /**
+     * Set platform details (or null for none).
+     *
+     * @param ?string $platform Platform details (or null for none)
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setPlatform(?string $platform): LoginTokenEntityInterface
+    {
+        $this->platform = $platform;
+        return $this;
+    }
+
+    /**
+     * Get platform details (or null for none).
+     *
+     * @return ?string
+     */
+    public function getPlatform(): ?string
+    {
+        return $this->platform;
+    }
+
+    /**
+     * Set expiration timestamp.
+     *
+     * @param int $expires Expiration timestamp
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setExpires(int $expires): LoginTokenEntityInterface
+    {
+        $this->expires = $expires;
+        return $this;
+    }
+
+    /**
+     * Get expiration timestamp.
+     *
+     * @return int
+     */
+    public function getExpires(): int
+    {
+        return $this->expires;
+    }
+
+    /**
+     * Set last session ID (or null for none).
+     *
+     * @param ?string $sid Last session ID (or null for none)
+     *
+     * @return LoginTokenEntityInterface
+     */
+    public function setLastSessionId(?string $sid): LoginTokenEntityInterface
+    {
+        $this->last_session_id = $sid;
```

Comment:
In this and the getter, `last_session_id` should be replaced with `lastSessionId` for consistency with the property defined in this class.